ui/client/containers/Processes.tsx

```diff
@@ -49,14 +56,16 @@ const ElementsRenderer: RowsRenderer = ({processes, getProcesses, statuses}) =>
   })
 }
 
-const sortable = ["name", "category", "modifyDate", "createdAt", "createdBy"]
+const sortable = ["name", "category", "modifyDate", "createdAt", "createdBy", "actionDate", "actionUser"]
 const filterable: Filterable = ["name", "processCategory", "createdBy"]
 const columns = [
   {key: "name", label: "Name"},
   {key: "category", label: "Category"},
   {key: "createdBy", label: "Created by"},
   {key: "createdAt", label: "Created at"},
   {key: "modifyDate", label: "Last modification"},
+  {key: "actionDate", label: "Last action date"},
```

Comment:
Is it understandable? I mean, one does not see what the action actually was? I know that it's somehow reflected in state, but I wonder if we can make it more clear? Wouldn't some tooltips be enough?

Reply:
Hmm, I can add some tooltip with information: Process was deploy/canceled/archived/unarchived at  / by?

Reply:
I was more thinking about tooltip with action data by status icon